Python-Function-LEN.md

```diff
@@ -0,0 +1,24 @@
+# Python len(x)
+
+`len()` is a built in function in Python 3. This method returns the length (the number of items) of an object. It takes one argument `x`. 
+
+## Arguments
+
+It takes one argument `x` - argument may be a sequence (such as a string, bytes, tuple, list, or range) or a collection (such as a dictionary, set, or frozen set).
+
+## Return Value
+
+This function returns the number of elements in the list
```

Comment:
Its according to the official documentation and that is why I have mentioned that "argument may be a sequence (such as a string, bytes, tuple, list, or range) or a collection (such as a dictionary, set, or frozen set)"